<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 30px;
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 gold, stop: 0.5 rgb(124, 0, 0), stop: 1.0 rgb(124, 0, 0));</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Di Gunakan Untuk</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCari">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>cari</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Data Kategori</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNamaKategori">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>Nama Kategori</string>
     </property>
    </widget>
    <widget class="QComboBox" name="txtDigunakanUntuk">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
color: rgb(0, 0, 0);
</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pelajar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mahasiswa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pekerja</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnSimpan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid goldenrod;
border-radius: 10px;
padding: 0 8px;
background-color: gold;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="gridKategori">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>341</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gold;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
gridline-color: rgb(124, 0, 0);</string>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 130px;
color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 gold, stop: 0.5 rgb(124, 0, 0), stop: 1.0 rgb(124, 0, 0));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>AZIZ MAULANA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string>200511084</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;ui/binkaibulet.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(&quot;ui/aziz.jpg&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid transparent;
background: transparent;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 gold, stop:0.497326 goldenrod, stop:1 red);</string>
    </property>
    <property name="text">
     <string>UNIVERSITAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>440</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid transparent;
background: transparent;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 gold, stop:0.497326 goldenrod, stop:1 red);</string>
    </property>
    <property name="text">
     <string>CIREBON</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_16">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-image: url(&quot;ui/um1.png&quot;);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid transparent;
background: transparent;
color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 gold, stop:0.497326 goldenrod, stop:1 red);</string>
    </property>
    <property name="text">
     <string>MUHAMMADIYAH</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>frame_4</zorder>
   <zorder>label_43</zorder>
   <zorder>label_42</zorder>
   <zorder>frame_16</zorder>
   <zorder>label_44</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
